docs/querying/aggregations.md

```diff
@@ -444,8 +445,8 @@ following can be the possible output of the aggregator
 | `["dim2"]`               | 2      | (10)                  |
 | `[]`                     | 3      | (11)                  |  
 
-As illustrated in above example, output number can be thought of as an unsigned n bit number where n is the number of dimensions passed to the aggregator. 
-The bit at position X is set in this number to 0 if a dimension at position X in input to aggregators is included in the sub-grouping. Otherwise, this bit 
+As illustrated in the example, the output number can be thought of as an unsigned _n_ bit number where _n_ is the number of dimensions passed to the aggregator. 
+The bit at position X is set in this number to 0 if a dimension at position X in the aggregator input is included in the sub-grouping. Otherwise, this bit 
```

Comment:
```suggestion
As the example illustrates, you can think of the output number as an unsigned _n_ bit number where _n_ is the number of dimensions passed to the aggregator. 
Druid sets the bit at position X for the number to 0 if the sub-grouping includes a dimension at position X in the aggregator input. Otherwise, Druid set this bit to 1. 
```

Reply:
Should be "Druid sets" in the last sentence!